### DISCUSSION TITLE
Support for defer and stream

### DISCUSSION THREAD

**Discussion by johann-su** [MichalLytek/type-graphql#1235] [Q&A]
maybe I just missed it but I couldn't find any reference to defer and stream operations [outlined in the apollo blog](https://www.apollographql.com/blog/community/backend/new-features-in-graphql-batch-defer-stream-live-and-subscribe/). 
Is that something type-graphql supports?

**Comment by MichalLytek** [comment]
Are they specified in official GraphQL spec?

They are client directives, so all the work has to be done by GraphQL engine like `graphql-js` that Apollo Server is using.
No special treatment by TypeGraphQL is needed.

**Reply by johann-su** [reply]
Ok so it seems that it is an ongoing discussion to add this to the spec.
There is a [branch](https://github.com/graphql/graphql-js/pull/2319) of graphql-js that adds support for this feature but it seems like this is still experimental.
